frontend/src/lib/components/apps/editor/component/default-codes.ts

```diff
@@ -1,11 +1,15 @@
 import type { AppComponent } from '.'
 
 export function defaultCode(component: string, language: string): string | undefined {
+	console.log(language)
```

Comment:
Hey, I saw this log and thought I should comment here just in case it was left unintentionally.

Reply:
Good catch!